lib/nat/structs.go

```diff
@@ -86,7 +86,7 @@ func (m *Mapping) OnChanged(subscribed MappingChangeSubscriber) {
 }
 
 func (m *Mapping) String() string {
-	return fmt.Sprintf("%s %s", m.protocol, m.address)
+	return fmt.Sprintf("%s/%s", m.protocol, m.address)
```

Comment:
I meant this, as used in other places:

```suggestion
	return fmt.Sprintf("%s/%s", m.address, m.protocol)
```